nautobot/docs/plugins/development.md

```diff
@@ -364,9 +364,10 @@ def leet_speak(input_str):
 
 This filter will then be available for use in computed field templates like so:
 
-```
+```python
```

Comment:
Probably:

````suggestion
```jinja2
````

Reply:
Updated, though currently there is no syntax highlighting for `jinja2`.